thavalon-webapp/src/components/gameComponents/roleInformation.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from "react";
+import { GameSocket, InboundMessage, InboundMessageType, OutboundMessageType } from "../../utils/GameSocket";
+import "../../styles/gameStyles/roleInformation.scss";
+
+interface RoleInfo {
+    abilities: string,
+    assassinatable: boolean,
+    description: string,
+    isAssassin: boolean,
+    otherInfo: string,
+    priorityTarget: string | undefined,
+    role: string,
+    seenPlayers: string[],
+    team: string,
+    teamMembers: string[],
+}
+
+interface Snapshot {
+    roleInfo: RoleInfo,
+    missions: string[],
+    log: string[],
+}
+
+export function RoleInformation(): JSX.Element {
+    const [roleInfoElement, setRoleInfoElement] = useState<JSX.Element>(<h1>Loading Role Info...</h1>);
+
+    /**
+     * Creates the JSX Element for representing the role information
+     * section of a game page.
+     * @param roleInfo The role info received in a snapshot message.
+     */
+    function createRoleInfoElement(roleInfo: RoleInfo): JSX.Element {
+        return (
+            <div id="playerInfo">
+                <h1 className="gameSectionHeader">Player Info</h1>
+                You are <span className={roleInfo.team.toLowerCase()}>{roleInfo.role}: {roleInfo.team.toUpperCase()}</span>
+                <br />
+                {roleInfo.description && <>
+                    <span className="roleDescription">{roleInfo.description}</span>
+                    <br />
+                </>}
+
+                {roleInfo.abilities && <>
+                    <span className="abilities">{roleInfo.abilities}</span>
+                    <br />
+                </>}
+
+                {roleInfo.seenPlayers.length > 0 && <>
+                    <span className="seenPlayers">You see {roleInfo.seenPlayers.join(", ")}</span>
+                    <br />
+                </>}
+
+                {roleInfo.team == "Evil" && roleInfo.teamMembers.length > 0 && <>
```

Comment:
General style question. Would this be easier to read as a bulleted list, rather than just new lines?